packages/frontend/src/state/positions/hooks.ts

```diff
@@ -108,6 +109,8 @@ export const useComputeSwaps = () => {
   const { getUsdAmt } = useUsdAmount()
   const { data } = useSwaps()
 
+  const dataSwapDep = JSON.stringify(data?.swaps.map((s) => pick(s, ['amount0', 'amount1', 'timestamp'])) ?? [])
```

Comment:
Thinking stringifying tx hash(id) alone would be enough as dep? cuz that would be unique value

Reply:
Agreed @adam5909099